includes/plugins/class-newspack-newsletters.php

```diff
@@ -1,446 +0,0 @@
-<?php
-/**
- * Newspack Newsletters integration class.
- *
- * @package Newspack
- */
-
-namespace Newspack;
-
-defined( 'ABSPATH' ) || exit;
-
-/**
- * Main class.
- */
-class Newspack_Newsletters {
-	const METADATA_DATE_FORMAT   = 'Y-m-d';
-	const METADATA_PREFIX        = 'NP_';
-	const METADATA_PREFIX_OPTION = '_newspack_metadata_prefix';
-
-	/**
-	 * The option name for choosing which metadata fields to sync.
-	 *
-	 * @var string
-	 */
-	const METADATA_FIELDS_OPTION = '_newspack_metadata_fields';
-
-	/**
-	 * Metadata keys map for Reader Activation.
-	 *
-	 * @var array
-	 */
-	public static $metadata_keys = [];
-
-	/**
-	 * Initialize hooks and filters.
-	 */
-	public static function init() {
-		\add_action( 'init', [ __CLASS__, 'setup_hooks' ], 15 );
-	}
-
-	/**
-	 * Setup hooks.
-	 */
-	public static function setup_hooks() {
-		\add_filter( 'newspack_newsletters_contact_data', [ __CLASS__, 'normalize_contact_data' ], 99 );
```

Comment:
This should no longer be applied as a filter inside the newsletters plugin. It should be called immediately after the `newspack_esp_sync_contact` and before the `::upsert()` call inside the `::sync()` method:

https://github.com/Automattic/newspack-plugin/blob/45534a692f81b280a5a5dfd83ddcbbe9362a5406/includes/reader-activation/sync/class-esp-sync.php#L93-L103

This improves readability and makes the sync logic more predictable.